Cineaste/Extension/SFSafariViewController+Style.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  SFSafariViewController+Style.swift
+//  Cineaste
+//
+//  Created by Felizia Bernutz on 03.07.18.
+//  Copyright © 2018 notimeforthat.org. All rights reserved.
+//
+
+import UIKit
+import SafariServices
+
+extension SFSafariViewController {
+    func addStyle() {
```

Comment:
We could also think of creating a custom SFSafariViewController as this extension is not a real extension to the SafariViewController. It does not supply any additional features to the existing Class.